desk/app/activity.hoon

```diff
@@ -247,11 +248,12 @@
         .^($-([ship nest] ?) %gx path)
       [test (~(put by checkers) group test)]
     =.  cor
+      ::  don't set channel default if group above it has setting
+      ?^  (~(get by area.volume) group)  cor
```

Comment:
```suggestion
      ?:  (~(has by area.volume) group)  cor
```

I would just `has:by` here if we're not using the result.

If the value from this gets set/copied elsewhere, may want to point to where that happens here. But not a big deal.